fusion-endpoint/pom.xml

```
@@ -17,7 +17,7 @@
         <javax.annotation.api.version>1.3.2</javax.annotation.api.version>
         <javaparser.version>3.23.0</javaparser.version>
         <swagger.codegen.version>3.0.27</swagger.codegen.version>
```

`swagger.codegen` and `swagger.codegen.generators` patch versions should probably go in sync. Swagger do not strictly follow semver, so there are elevated chances to have issues.

```suggestion
        <swagger.codegen.version>3.0.20</swagger.codegen.version>
```